pkg/fanal/analyzer/os/redhatbase/centos_test.go

```diff
@@ -36,6 +36,9 @@ func Test_centosOSAnalyzer_Analyze(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			a := centOSAnalyzer{}
 			f, err := os.Open(tt.inputFile)
+			if err != nil {
+				return
+			}
```

Comment:
I think we can just move `require.NoError` below here instead of the error handling.